.github/workflows/someip2val_build.yml

```diff
@@ -114,10 +114,8 @@ jobs:
           images: |
             ghcr.io/${{ github.repository }}/someip-feeder
           tags: |
-            type=sha
-            type=ref,event=tag
-            type=edge
-            type=raw,value=latest,enable={{is_default_branch}}
+            type=ref,event=branch
```

Comment:
@int0x27 @SebastianSchildt

If we change like this we get the same setup as we use for Databroker and DBCFeeder. Please check if this would cause any problems

Reply:
Looks OK, we don't need sha, latest and edge.
Btw I found a regression happenning from: https://github.com/eclipse/kuksa.val.feeders/pkgs/container/kuksa.val.feeders%2Fsomeip-feeder/96935519?tag=sha-0007f8c
It seems docker image contains 3rd unknown architecture